### ISSUE TITLE
Can some one please provide example?

### ISSUE THREAD

**Issue by tim-shand** [schakra/Active-Directory-Sample-for-PHP-Developers#2]
Hi Team, I am having issues getting this configured - absolute newbie at this and have been tasked to create a custom web portal to display SQL data etc. One of the requirements is that I have to use ADFS authentication. Can someone please show me their config so I can compare with my own? 

Cheers, Tim.

**Comment by Pukky01** [comment]
Hi, did you manage to do that? I have similar problem.

**Comment by tim-shand** [comment]
Yes, all sorted and working. Issue was on the ADFS server. Follow the steps and you should be fine. I also used this guide: https://groups.google.com/forum/#!topic/simplesamlphp/I8IiDpeKSvY

**Comment by tim-shand** [comment]
And this one is pretty good: https://confluence.seges.dk/display/PUB/SimpleSamlPhp

**Comment by sathyamlokare** [comment]
hi @ts91nz I am facing an issue while implementing this
https://github.com/schakra/Active-Directory-Sample-for-PHP-Developers/issues/3

Can you please help me with this ? any links , examples will be really helpful

Thanks
